crypto/crypto.go

```diff
@@ -20,8 +22,38 @@ func GetPubKeyHex(privateKey PrivateKey) string {
 	return hex.EncodeToString(privKey.PublicKey.X) + hex.EncodeToString(privKey.PublicKey.Y)
 }
 
-func GeneratePrivateKey() (PrivateKey, error) {
-	return elliptic.GeneratePrivateKey(elliptic.Secp256k1)
+func GeneratePrivateKey(difficulty int) PrivateKey {
+	for true {
+		privKey, err := elliptic.GeneratePrivateKey(elliptic.Secp256k1)
+		if err != nil {
+			fmt.Println("Error while generating private key", err)
```

Comment:
remove this `Println` and make it return `err` as it used to

Reply:
Done